lib/seattleflu/id3c/cli/command/etl/clinical.py

```diff
@@ -286,6 +304,115 @@ def flu_shot(flu_shot_response: Optional[Any]) -> list:
 
     return [flu_shot_map[flu_shot_response]]
 
+
+def covid_shot(covid_shot_response: Optional[Any]) -> list:
+    """
+    Given a *covid_shot_response*, returns yes/no value for CovidShot key.
+
+    >>> covid_shot(0.0)
+    ['no']
+
+    >>> covid_shot('TRUE')
+    ['yes']
+
+    >>> covid_shot('maybe')
+    Traceback (most recent call last):
+        ...
+    id3c.cli.command.etl.UnknownCovidShotResponseError: Unknown COVID shot response «maybe»
+
+    """
+    if covid_shot_response is None:
+        LOG.debug("No COVID shot response found.")
+        return [None]
+
+    if isinstance(covid_shot_response, str):
+        covid_shot_response = covid_shot_response.lower()
+
+    covid_shot_map = {
+        0.0 : "no",
+        1.0 : "yes",
+        "false": "no",
+        "true": "yes",
+    }
+
+    if covid_shot_response not in covid_shot_map:
+        raise UnknownCovidShotResponseError(
+            f"Unknown COVID shot response «{covid_shot_response}»")
+
+    return [covid_shot_map[covid_shot_response]]
+
+
+def covid_shot_maunufacturer(covid_shot_manufacturer_name: Optional[Any]) -> list:
+    """
+    Given a *covid_shot_manufacturer_name*, returns validated and standarized value.
+
+    >>> covid_shot_maunufacturer('PFIZER')
+    ['pfizer']
+
+    >>> covid_shot_maunufacturer('Moderna')
+    ['moderna']
+
+    >>> covid_shot_maunufacturer('SomeCompany')
+    Traceback (most recent call last):
+        ...
+    id3c.cli.command.etl.UnknownCovidShotManufacturerError: Unknown COVID shot manufacturer «somecompany»
+
+    """
+    if covid_shot_manufacturer_name is None:
+        LOG.debug("No COVID shot manufacturer name found.")
+        return [None]
+
+    if isinstance(covid_shot_manufacturer_name, str):
+        covid_shot_manufacturer_name = covid_shot_manufacturer_name.lower().strip()
+
+    valid_covid_manufacturers = [
+        "pfizer",
+        "moderna",
+    ]
```

Comment:
Seems like this list might need to expand later, but raising the warning as below should clue us in if we need to update this. :thumb: